packages/story-editor/src/components/checklist/checks/videoOptimization.js

```diff
@@ -246,7 +273,10 @@ const BulkVideoOptimization = () => {
           <Thumbnail
             key={element.resource.id}
             onClick={handleClickThumbnail(element)}
-            isLoading={element.resource.isTranscoding}
+            isLoading={
+              isResourceTranscoding(element.resource.id) ||
```

Comment:
Q: Why not change it to `isNewResourceTranscoding` or something similar if it's always used for new resources only?

It's still a bit unclear to differentiate between `isResource...` and `isCurrentResource...`.

Alternatively, maybe let's make the function comments a bit more detailed? E.g.
`Is the current resource being processed.` -> it would be good to mention that this is the original resource that's used for generating the new resource. Thoughts?

Reply:
This change does make things more confusing for sure. But to untangle this, would mean writing lots of this code. Refactoring for readability with no part of the scope of this change. I would like I can add comments and change function names. Refactoring of this code to make is clearer, should be done in another follow up ticket.

Reply:
Updating just the comments sounds good to me for now as well!

Reply:
I have some local changes in this regard ready; will create a PR for this in a bit.